.github/workflows/auto-review-trigger.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - synchronize
+  pull_request_review:
+    types:
+      - submitted
+      - dismissed
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: Pull Request Number
+        type: string
+        required: true
+
+name: Auto Review Bot Trigger
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    name: Pull Request
+    if: github.event_name == 'pull_request_target'
+    steps:
+      - name: Write PR Number
+        run: echo $PR_NUMBER > pr-number.txt
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+
+      - name: Save PR Number
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8
+        with:
+          name: pr-number
+          path: pr-number.txt
+  
+  pr-review:
+    runs-on: ubuntu-latest
+    name: Review
+    if: github.event_name == 'pull_request_review' && github.event.review.sender != 'eth-bot' && (github.event.review.state == 'approved' || github.event.review.state == 'dismissed')
+    steps:
+      - name: Write PR Number
+        run: echo $PR_NUMBER > pr-number.txt
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+
+      - name: Save PR Number
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8
+        with:
+          name: pr-number
+          path: pr-number.txt
```

Comment:
Why do two jobs upload the artifact?

Reply:
There are actually three jobs, one for each type. I could make it all one job, but it would get messy really quick.

Reply:
It would not be easier for the bot to ask for the EIP number desired (and confirmed it is not used) and >=2  Editors can assign it ? 
Changing .yml's are harder to test, this is why the original .yml code is simple,  therefore we code the bot for whatever the Editors wish.

Reply:
@JEAlfonsoP It's actually a bit more subtle than that. There's an issue with the workflow files themselves that is not resolvable within the scope of the EIP-Bot repository. However, EIP-Bot will continue to work as if nothing happened.